doc/articles/features/using-the-uno-sdk.md

```diff
@@ -241,3 +241,20 @@ By Default when using the Uno.Sdk you get the added benefit of default includes
 - `*.Android.cs`
 
 As discussed above setting `EnableDefaultUnoItems` to false will disable these includes.
+
+## Apple Privacy Manifest Support
+
+Apple is introducing a [Privacy Manifest file](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) (named `PrivacyInfo.xcprivacy`) that needs to be included as of May 1st 2024.
+
+The Uno.Sdk (5.2 or later) automatically includes the `Platforms/iOS/PrivacyInfo.xcprivacy` file in the app bundle. You can find an example of this file in the [Uno.Templates](https://aka.platform.uno/apple-privacy-manifest-sample) repository
+
+For more information on how to include privacy entries in this file, see the [Microsoft .NET documentation](https://learn.microsoft.com/en-us/dotnet/maui/ios/privacy-manifest) on the subject, as well as [Apple's documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
+
+> [!NOTE]
+> If your application is not using the Uno Platform 5.1 or earlier, you can include the file using the following:
```

Comment:
Typo?

```suggestion
> If your application is using the Uno Platform 5.1 or earlier, you can include the file using the following:
```

Reply:
I think we may need to re-word this for people who may be using newer versions of Uno but without using the Uno.Sdk?

Reply:
Indeed, I'll adjust.